packages/shared-pure/src/types/EthereumAddress.test.ts

```diff
@@ -68,4 +68,13 @@ describe(EthereumAddress.name, () => {
       '0x0000000000000000000000000001234567890AbC',
     )
   })
+
+  it('properly checks ignoring case', () => {
```

Comment:
This seems like it's not necessary since the default implmentation ignores casing if it's all lower/upper

Reply:
That's no my experience. Using `EthereumAddress.check` with incorrect casing returns false. That's why I added `checkIgnoringCase()`